src/NHSD.BuyingCatalogue.Identity.Api/appsettings.json

```diff
@@ -1,5 +1,10 @@
 {
   "issuerUrl": "http://docker.for.win.localhost:5102",
```

Comment:
Although this hasn't changed, the inclusion of `docker.for.win` here causes the `UserInfo` endpoint to fail when running in VS.

Reply:
switched out to localhost for now.

Reply:
oops. Obviously forgot to save it. pushed now.